src/wiki/ventoy.md

```diff
@@ -0,0 +1,131 @@
+# Creating a Multi-OS Bootable Drive (with Ventoy)
+In this article, you will learn how to create a multi-OS bootable drive. 
+
+Having a bootable USB at all times can help you when you need to reinstall an OS, and a multi-OS bootable drive can allow you to install different OS without having to reformat a drive.
+
+## Creation of the USB
+This section will teach you how to create a bootable drive for your needs.
+
+::: danger
+Installing Ventoy to create a bootable drive **will wipe all your data from that drive**. Back up your data on the drive before proceeding.
+:::
+1. Download Ventoy [here](https://www.ventoy.net/en/download.html). This is the tool that we will be using to create a bootable drive.
+![Download page of Gparted](img/ventoy/Gparted.png)
+2. Plug in a USB drive that has at least 8GB.
+3. Right click on the file that you have just downloaded and select extract. Then, enter the folder named `ventoy-1.0.xx` and open an application named `Ventoy2Disk.exe`. A UAC prompt will pop-up, click yes.
+4. Select a drive in the device selection section. This will be your bootable drive. If you can't see the drive you want to use, press the green button to refresh the list.
+![Select USB](<img/ventoy/Ventoy screen.png>)
+5. And the top right of the program click on options and get sure that the secure boot option is enabled and in partition stile select “GPT”.
+![Menu for selecting settings in Ventoy](<img/ventoy/Secure boot and GPT.png>)
+6. Click install, and Ventoy will create a bootable drive for you. After it finishes, you will see a drive named as "Ventoy", this is where you will place your ISO files for the bootable drive.
+![Click on Install button](img/ventoy/Install.png)
+
+## Downloading Windows to the Bootable Drive
+### Downloading the ISO file
+
+1. For Windows 10, go to [https://www.microsoft.com/software-download/windows10](https://www.microsoft.com/software-download/windows10) <br>
+To download Windows 11 (Recommended), head to [https://www.microsoft.com/software-download/windows11](https://www.microsoft.com/software-download/windows11)
```

Comment:
If Windows 11 is recommended, why not put that first?

Also this entire section duplicates https://msft.chat/wiki/downloading-windows, just link there and mention the differences here.